src/configs/elixir.rs

```diff
@@ -1,24 +1,17 @@
-use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+use crate::config::{ModuleConfig, RootModuleConfig};
 
-use ansi_term::{Color, Style};
 use starship_module_config_derive::ModuleConfig;
 
 #[derive(Clone, ModuleConfig)]
 pub struct ElixirConfig<'a> {
-    pub symbol: SegmentConfig<'a>,
-    pub version: SegmentConfig<'a>,
-    pub otp_version: SegmentConfig<'a>,
-    pub style: Style,
```

Comment:
We are keeping the `symbol` and `style` configuration options. Can you add these back please.